### ISSUE TITLE
V0.6.2 : Sur la fond

### ISSUE THREAD

**Issue by hgaden** [sumaris-net/sumaris-app#10]
- l'heure de vente est-elle nécessite-t-elle d'être un champ obligatoire ?
-

**Comment by blavenie** [comment]
Non, normalement l'heure n'est obligatoire que si le lieu de vente est obligatoire.
Mais effectivement l'affichage "en rouge" n'est pas bon

**Comment by hgaden** [comment]
Ce n'est pas qu'une histoire de couleur, car si l'heure n'est pas remplie, on ne peut pas sauvegarder.

**Comment by hgaden** [comment]
> Non, normalement l'heure n'est obligatoire que si le lieu de vente est obligatoire.
> Mais effectivement l'affichage "en rouge" n'est pas bon

Ce n'est pas qu'une histoire de couleur apparemment, car si l'heure n'est pas remplie, on ne peut pas sauvegarder.
